system-addon/lib/TopSitesFeed.jsm

```diff
@@ -105,6 +107,9 @@ this.TopSitesFeed = class TopSitesFeed {
       data: this._getPinnedWithData()
     }));
   }
+  saveToPocket(action) {
```

Comment:
We can actually remove the pocket stuff in the TopSitesFeed here - since we don't want pocket in top sites anymore